### PR TITLE
Fixed full description to be available upon rendering

### PR DESCRIPTION
**Pull request by Dantheman720**
Fixed so that full_description can be seen on rendering.

### CODE REVIEW

**Comment by LightGuard** [comment]
The changes to description happened both in YouTube and Vimeo, correct?

**Comment by LightGuard** [comment]
I think I'd like to see `description` changed so that if we're doing a full description we just return the description and not worry about length.

**Comment by Dantheman720** [comment]
Definitely seems less error prone that way.  I'll make the fix.

As for the changes to description, yeah, they happen in both.  It's in video_base.

**Comment by paulrobinson** [comment]
Can this be merged now?

**Comment by wesbos** [comment]
@LightGuard Please merge this in - we just passed our tests over at https://github.com/redhat-developer/developers.redhat.com/pull/496

File: lib/aweplug/helpers/video/video_base.rb [hunk 1]
```diff
@@ -39,32 +39,30 @@ def initialize(video, site, default_ttl = 86400) # A day seems good for videos
...
                 break
-              else
-                out += s
               end
             end
             # Deal with the case that the description has no sentence end in it
             out = (out.empty? || out.length < 60) ? d : out
           end
-          out = out.gsub("\n", ' ')[0..max_length]
+          out = out.gsub("\n", ' ')[0..150]
```
Comment (LightGuard) [inline lib/aweplug/helpers/video/video_base.rb]: This should still be max_length, and line 47 should be move up outside of that if block

**Comment by LightGuard** [comment]
Great, thanks @Dantheman720. @wesbos This is in now.

**Pull request merged**